tests/cucumber/features/step_definitions/accounts_steps.coffee

```diff
@@ -0,0 +1,66 @@
+do ->
+  'use strict'
+
+  _ = require('underscore')
+
+  module.exports = ->
+
+    url = require('url')
+
+    @Given /^I am a new user$/, ->
+      @server.call('reset')
+
+    @When /^I navigate to "([^"]*)"$/, (relativePath, callback) ->
+      @client
+        .url(url.resolve(process.env.ROOT_URL, relativePath))
+        .call(callback)
```

Comment:
I vote that we move it to a shared file the first time one of us needs to use it in a new place, instead of doing that before we need to.